ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```
@@ -78,6 +78,8 @@ conformance_test_tool_args = [
         "TLSOnePointThreeIT",
         "TLSAtLeastOnePointTwoIT",
         "CommandDeduplicationPeriodValidationIT:OffsetPruned",  # requires pruning not available in canton community
+        "ActiveContractsServiceIT:AcsBeforePruningOffsetIsDisallowed",  # requires pruning not available in canton community
```

Could you add a link to the issue ? 

```suggestion
        # TODO https://github.com/DACH-NY/canton/issues/1126
        #  reactive the two following test once the issue is fixed.
        "ActiveContractsServiceIT:AcsBeforePruningOffsetIsDisallowed",  # requires pruning not available in canton community
```